pkg/commands/option/remote.go

```diff
@@ -0,0 +1,75 @@
+package option
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+	"go.uber.org/zap"
+)
+
+// RemoteOption holds options for client/server
+type RemoteOption struct {
+	RemoteAddr    string
+	customHeaders []string
+	token         string
+	tokenHeader   string
+	remote        string // deprecated
+
+	// this field is populated in Init()
+	CustomHeaders http.Header
+}
+
+func NewRemoteOption(c *cli.Context) RemoteOption {
+	r := RemoteOption{
+		RemoteAddr:    c.String("server"),
+		customHeaders: c.StringSlice("custom-headers"),
+		token:         c.String("token"),
+		tokenHeader:   c.String("token-header"),
+		remote:        c.String("remote"), // deprecated
+	}
+
+	return r
+}
+
+// Init initialize the options for client/server mode
+func (c *RemoteOption) Init(logger *zap.SugaredLogger) (err error) {
```

Comment:
Looks like we don't need to return an error.

Reply:
good catch! removed.